examples/bring_your_own_fips202_static/README.md

```diff
@@ -1,28 +1,56 @@
 [//]: # (SPDX-License-Identifier: CC-BY-4.0)
 
-# Bring your own FIPS-202 (Static State Variant)
+# Bring Your Own FIPS-202 (Static State Variant)
 
-This directory contains a minimal example for how to use mlkem-native with external FIPS202
-HW/SW-implementations that use a single global state (for example, some hardware accelerators).
-Specifically, this example demonstrates the use of the serial-only FIPS-202 configuration
-`MLK_CONFIG_SERIAL_FIPS202_ONLY`.
+This directory contains a minimal example for using mlkem-native with a custom FIPS-202 implementation
+that uses a single global state. This is common for hardware accelerators that can only hold one
+Keccak state at a time.
+
+## Use Case
+
+Use this approach when:
+- Your application already has a FIPS-202 software/hardware implementation you want to reuse
+- Your FIPS-202 implementation does not support multiple active SHA3/SHAKE computations.
 
 ## Components
 
-An application using mlkem-native with a custom FIPS-202 implementation needs the following:
+1. Arithmetic part of mlkem-native: [`mlkem/src/`](../../mlkem/src) (excluding `fips202/`)
+2. A secure random number generator implementing [`randombytes.h`](../../mlkem/src/randombytes.h)
+3. Custom FIPS-202 implementation with header compatible with [`fips202.h`](../../mlkem/src/fips202/fips202.h)
+4. Stub `fips202x4.h` header (the 4x functions are unused in this mode)
```

Comment:
This isn't entirely true. `mlk_shake256x4` is still used.
I have opened https://github.com/pq-code-package/mlkem-native/issues/1274 for that. Once that is implemented one does not need to provide the `fips202x4.h` header anymore.

Reply:
Ack